docs/index.md

```diff
@@ -1,7 +1,7 @@
-<p style="text-align: center;">
-    <img src="assets\images\Obscuro-white-strip.png">
+<p style="text-align: left;" class="build">
+    <img src="assets\images\buildwithprivacy.png">
 </p>
 
-Obscuro is a decentralised Ethereum Layer 2 Rollup protocol with privacy.
+Welcome to Obscuro. Privacy for Ethereum on an L2 roll-up. No SDKs, 100% EVM.
```

Comment:
Is "An L2 for privacy on Ethereum" better?

And are we assuming too much to assume that people know what an L2 is? Should we spell out "layer 2"?